sjsonnet.StaticError: Can't use self outside of an object
    at [Self].(sjsonnet/test/resources/test_suite/error.static_error_self.jsonnet:17:2)

